contracts/LockProvider/LockDealProvider.sol

```diff
@@ -52,6 +52,8 @@ contract LockDealProvider is BasicProvider, LockDealState {
         uint256 poolId,
         uint256[] memory params
     ) internal override {
+        require(block.timestamp <= params[1], "invalid startTime");
```

Comment:
maybe we can do it as :
```
if (block.timestamp > params[1])
    startTimes[poolId] = params[1];
```

Reply:
Do we allow the invalid start time? I think the wrong param should be reverted.

Reply:
I'm fine both ways.